diff --git a/node_modules/expo-structured-headers/android/build.gradle b/node_modules/expo-structured-headers/android/build.gradle
index 487d805..9989f9d 100644
--- a/node_modules/expo-structured-headers/android/build.gradle
+++ b/node_modules/expo-structured-headers/android/build.gradle
@@ -19,7 +19,7 @@ android {
 }
 
 dependencies {
-  implementation "androidx.appcompat:appcompat:1.2.0"
+  implementation "androidx.appcompat:appcompat:1.6.1"
 
   testImplementation 'junit:junit:4.13.2'
   testImplementation 'commons-codec:commons-codec:1.14'
